packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/container.ts

```diff
@@ -93,6 +108,30 @@ export class Container extends ServiceExtension {
       containerProps = hookProvider.mutateContainerDefinition(containerProps);
     });
 
+    // If no observability extensions have been added to the service description, we can configure
+    // default awslogs log driver
+    if (!containerProps.logging) {
+      // Create a log group for the service, if one is not provided by the user
+      if (!this.logGroup) {
+        this.logGroup = new awslogs.LogGroup(this.scope, `${this.parentService.id}-logs`, {
```

Comment:
Is this is a default policy configurations or best practices mentioned somewhere?

Reply:
Actually the CDK default log group configuration would retain the logs for 2 years, it would be wasteful and would incur unnecessary costs while the most common use case for these logs would be to help in troubleshooting. I couldn't find a best practices doc but chose these values looking at precursors (like Copilot). Also, the user can provide a log group if they want to customize these values!